### PR TITLE
MSSQL: fix tests

### PR DESCRIPTION
**Pull request by alexkart**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  |

### CODE REVIEW

File: framework/CHANGELOG.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ Yii Framework 2 Change Log
 2.0.23 under development
 ------------------------
 
-- Bug #17413, #17418: Fixed MSSQL tests (alexkart)
-- Bug #17413: Fixed MSSQL tests: `QueryBuilderTest::testGetColumnType()`, `QueryBuilderTest::testResetSequence()` (alexkart)
+- Bug #17413, #17418, #17426, #17431: Fixed MSSQL tests (alexkart)
 - Bug #17420: Fixed loading of column default values for MSSQL (alexkart)
+- Bug #17395: Fixed issues with actions that contain underscores in their names (alexkart)
```
Comment (samdark) [inline framework/CHANGELOG.md]: I don't think this one should be moved.

File: tests/framework/db/mssql/QueryBuilderTest.php [hunk 2]
```diff
@@ -85,9 +86,8 @@ public function testCommentColumn()
 
     public function testCommentTable()
     {
-        if ($this->driverName === 'sqlsrv') {
-            $this->markTestSkipped('Should be fixed');
-        }
+        // related to https://github.com/yiisoft/yii2/pull/17364
+        $this->markTestSkipped('Should be fixed');
```
Comment (samdark) [inline tests/framework/db/mssql/QueryBuilderTest.php]: Looks like tests for non-MSSQL got disabled. Is that intentional?

File: framework/CHANGELOG.md (hunk 1 above)
Comment (alexkart) [reply framework/CHANGELOG.md]: Why? It was merged after 2.0.22 release. Am I wrong?

File: tests/framework/db/mssql/QueryBuilderTest.php (hunk 2 above)
Comment (alexkart) [reply tests/framework/db/mssql/QueryBuilderTest.php]: No, that condition was unnecessary, it's already MSSQL specific test class.

File: framework/CHANGELOG.md (hunk 1 above)
Comment (samdark) [reply framework/CHANGELOG.md]: Ah, right!

**Comment by samdark** [comment]
Merged. Thanks!

**Pull request merged**